src/utils/spp.ts

```diff
@@ -29,5 +30,45 @@ export const spp = {
     if (response.WebTemplateConfiguration !== 'CONTENTCTR#0') {
       throw Error(`${siteUrl} is not a content site.`);
     }
+  },
+  /**
+   * Gets a SharePoint Premium model by title or unique id
+   * @param contentCenterUrl a content center site URL
+   * @param title model title
+   * @param id model unique id
+   * @returns SharePoint Premium model
+   */
+
+  async getModel(contentCenterUrl: string, title?: string, id?: string): Promise<SppModel> {
```

Comment:
Personally, I'm more a fan of splitting this into two functions instead of having a 'useless' parameter.
`getModelById` and `getModelByTitle` would cover the load better.